examples/cpu_render.rs

```diff
@@ -14,32 +14,23 @@ fn main() {
     const TREE_SIZE: u32 = 64;
     let viewport_size_width = 150;
     let viewport_size_height = 150;
-    let mut tree = shocovox_rs::octree::Octree::<Albedo, BRICK_DIMENSION>::new(TREE_SIZE)
-        .ok()
-        .unwrap();
-
-    tree.insert(&V3c::new(1, 3, 3), voxel_color)
-        .expect("insert of voxel to work");
-    for x in 0..TREE_SIZE {
-        for y in 0..TREE_SIZE {
-            for z in 0..TREE_SIZE {
-                if ((x < (TREE_SIZE / 4) || y < (TREE_SIZE / 4) || z < (TREE_SIZE / 4))
-                    && (0 == x % 2 && 0 == y % 4 && 0 == z % 2))
-                    || ((TREE_SIZE / 2) <= x && (TREE_SIZE / 2) <= y && (TREE_SIZE / 2) <= z)
-                {
-                    tree.insert(
-                        &V3c::new(x, y, z),
-                        Albedo::default()
-                            .with_red((255 as f32 * x as f32 / TREE_SIZE as f32) as u8)
-                            .with_green((255 as f32 * y as f32 / TREE_SIZE as f32) as u8)
-                            .with_blue((255 as f32 * z as f32 / TREE_SIZE as f32) as u8)
-                            .with_alpha(255),
-                    )
-                    .ok()
-                    .unwrap();
-                }
-            }
-        }
+    // let mut tree = shocovox_rs::octree::Octree::<Albedo, BRICK_DIMENSION>::new(TREE_SIZE)
+    //     .ok()
+    //     .unwrap();
+
+    let tree;
+    if std::path::Path::new("example_junk_minecraft_tree").exists() {
```

Comment:
maybe `let example_name = "example_junk_minecraft_tree"`?

Reply:
oh no, I forgot to revert the changes in this file... :facepalm: